core/src/Kokkos_Macros.hpp

```diff
@@ -154,13 +154,16 @@
 #define KOKKOS_COMPILER_IBM __IBMCPP__
 #elif defined(__IBMC__)
 #define KOKKOS_COMPILER_IBM __IBMC__
+#elif defined(__ibmxl_vrm__)  // xlclang++
```

Comment:
These changes did not work for me (cuda/10.1 builds failed), I needed the macros as below to get a successful cuda/10.1+xl/16.1.1 build:

```bash
__ibmxl_vrm__ -> __ibm__
```

Reply:
I can't find the documentation for `__ibm__`
I was looking AT https://www.ibm.com/support/knowledgecenter/SSGH3R_16.1.0/com.ibm.xlcpp161.aix.doc/compiler_ref/xlmacros.html

Reply:
Updated by commit 874d8c2

Reply:
For the record `__ibmxl_vrm__` works as expected on Summit.